.github/workflows/main.yml

```diff
@@ -53,7 +53,7 @@ jobs:
        run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
 
     - name: Bazel lint check
-      run: ./buildifier --lint=warn --mode=check --warnings=all -r app data domain model testing utility tools BUILD.bazel WORKSPACE oppia_android_test.bzl
+      run: ./buildifier --lint=warn --mode=check --warnings=-native-android,+out-of-order-load,+unsorted-dict-items -r app data domain model testing utility tools BUILD.bazel WORKSPACE oppia_android_test.bzl && echo "Bazel lint check completed successfully"
```

Comment:
Could we maybe just call our buildifier script here instead? Having a single source of truth for linting configuration is preferable. Ditto for the others (but not in this PR).

Reply:
done

Reply:
Is this done? It seems like we're still calling buildifier directly (also note that 'not in this PR' meant not to move the other linters in this PR--we should still update the buildifier use to reference the script directly.

Reply:
we will be introducing script with the pre-push. As of now, I think we are doo to do with the command in the main.yml

Reply:
To clarify: I'm actually suggesting we move to the pattern of running our linters via scripts to avoid duplicating the code between the workflow file & the script. I think adding the pre-push hook in a subsequent PR is fine, but can we also move this to a linter script in this PR to establish that new pattern?